### ISSUE TITLE
installing with npm - Unmet peer dependcy

### ISSUE THREAD

**Issue by louisdoe** [driftyco/ionic-app-scripts#845]
#### Short description of the problem:
I installed this using : `npm install @ionic/app-scripts@latest --save-dev`
and now I have a dependency error and can't use `ionic serve` anymore.

#### What behavior are you expecting?

**Steps to reproduce:**
1. npm install @ionic/app-scripts@latest --save-dev
2. ionic serve

```
insert any relevant code between the above and below backticks
```

**Which @ionic/app-scripts version are you using?**
version:
@ionic/app-scripts@1.2.1 

**Other information:** (e.g. stacktraces, related issues, suggestions how to fix, stackoverflow links, forum links, etc)

**Comment by danbucholtz** [comment]
What is the error?

Thanks,
Dan

**Comment by louisdoe** [comment]
Actually, I reverted back to the version initially installed with the rest of the ionic package (which is different than the lastest version) and now it's good.
I think it needed to be version @ionic/app-scripts@1.1.4

**Comment by danbucholtz** [comment]
This is fixed in `1.2.3`.

Thanks,
Da
